Binary files PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab.orig/.git/index and PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab/.git/index differ
diff -rupN PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab.orig/SDL/SDLMain.cpp PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab/SDL/SDLMain.cpp
--- PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab.orig/SDL/SDLMain.cpp	2020-11-23 11:24:47.893402871 -0500
+++ PPSSPPSDL-087de849bdc74205dd00d8e6e11ba17a591213ab/SDL/SDLMain.cpp	2020-11-28 09:39:23.004618567 -0500
@@ -581,13 +581,13 @@ int main(int argc, char *argv[]) {
 #endif
 
 	if (mode & SDL_WINDOW_FULLSCREEN_DESKTOP) {
-		pixel_xres = g_DesktopWidth;
-		pixel_yres = g_DesktopHeight;
+		pixel_xres = 480;
+		pixel_yres = 320;
 		g_Config.bFullScreen = true;
 	} else {
 		// set a sensible default resolution (2x)
-		pixel_xres = 480 * 2 * set_scale;
-		pixel_yres = 272 * 2 * set_scale;
+		pixel_xres = 480;
+		pixel_yres = 320;
 		if (portrait) {
 			std::swap(pixel_xres, pixel_yres);
 		}
